MoreLinq.Test/SkipLastWhileTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿#region License and Terms
+// MoreLINQ - Extensions to LINQ to Objects
+// Copyright (c) 2024 Andy Romero (armorynode). All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#endregion
+
+namespace MoreLinq.Test
+{
+    using System.Collections.Generic;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class SkipLastWhileTest
+    {
+        [Test]
```

Comment:
Add a test case to demonstrate that `SkipWhileLast` is lazy execution semantics, like so:

```suggestion
        [Test]
        public void IsLazy()
        {
            _ = new BreakingSequence<object>().SkipLastWhile(BreakingFunc.Of<object, bool>());
        }

        [Test]
```